### PR TITLE
Added Senlab M

### PR DESCRIPTION
**Pull request by J2thatsme**

### CODE REVIEW

File: types/sensingLabs/senlabM/default.schema.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "processingType": "uplink_decoder",
+  "topic": "default",
+  "title": "Default",
+  "properties": {
+    "pulse": {
+      "title": "Pulse",
+      "type": "integer",
+      "description": "Total praw ulses gotten"
```
Comment (mariusgiger) [inline types/sensingLabs/senlabM/default.schema.json:12]: please check spelling

File: types/sensingLabs/senlabM/default.schema.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    },
+    "wireCutStatus": {
+      "title": "Wire cut status",
+      "type": "integer",
+      "description": "Wire cut status"
+    },
+    "pulse1": {
+      "title": "Pulse 1",
+      "type": "integer",
+      "description": "Total raw pulses with first node gotten"
```
Comment (mariusgiger) [inline types/sensingLabs/senlabM/default.schema.json:22]: gotten -> received?

File: types/sensingLabs/senlabM/meta.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  "author": "Akenza AG",
+  "firmwareVersion": "V1.0.0",
+  "loraDeviceClass": "A",
+  "availableSensors": [
+    "Pulse"
+  ],
+  "outputTopics": [
+    "default",
+    "lifecycle",
+    "start_up"
```
Comment (mariusgiger) [inline types/sensingLabs/senlabM/meta.json:16]: rename to boot or system?

File: types/sensingLabs/senlabM/uplink.spec.js [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+        assert.equal(value.data.redundancyFactor, 1);
+        assert.equal(value.data.txPeriod, 1800);
+
+        utils.validateSchema(value.data, startupSchema, { throwError: true });
+      });
+
+      consume(data);
+    });
+
+    it("Should decode Senlab SenlabM payload", () => {
```
Comment (mariusgiger) [inline types/sensingLabs/senlabM/uplink.spec.js:71]: also test the conversion with custom fields

**Review by mariusgiger — COMMENTED** [review]
looks good, a bit of feedback

**Review by sevyakenza — APPROVED** [review]

**Pull request merged**